src/systems/filecoin_blockchain/storage_power_consensus/_index.md

```diff
@@ -38,8 +39,9 @@ Per the above, we also clearly distinguish putting storage on-chain from gaining
 
 Thereafter, power can be either *active* or *inactive* as defined in {{<sref storage_mining_subsystem>}}.
 
-In-deal or not does not determine whether power is active or inactive. Power has to come from active in-deal data. Active Power comes from deal data in Active sectors (and hence the deals are also active). Inactive Power comes from data in Committed, Recovering and Failing sectors. Miners still need to maintain pledge collateral for Inactive Power.
+It is important to remember that storage contributing to power and storage state are orthogonal notions. Specifically, power comes from storage being used in live-deals in active sectors (as opposed to non-deal active storage or in-deal inactive storage).
 
+Miners need to maintain pledge collateral for both active and inactive in-deal storage.
```

Comment:
Blocking: lets not create a new concept here called in-deal storage? The previous paragraph here seems fine for now, we might need to update this again in the next few days.